database/leveldb/db.go

```diff
@@ -220,3 +227,13 @@ func updateError(err error) error {
 		return err
 	}
 }
+
+func copyBytes(bytes []byte) []byte {
```

Comment:
We already have a `copyBytes` function in `database/encdb/db.go`
Let's move this to `utils`

Reply:
done